src/CommunityVoices/App/Website/View/Quote.php

```diff
@@ -283,6 +283,15 @@ public function getQuoteUpload($request, $errors = self::ERRORS_DEFAULT)
                 '<form>' . $this->transcriber->toXml($form) . '</form>'
             );
         }
+        // booleans do not work in parsing to xml, requiring this ugly thing
+        if($request->request->has('batch_submit') && empty($errors->upload->errors)) {
```

Comment:
What exactly is the purpose in renaming this?  You can make the `input` attached to the submit button named `repeatedQuote` and it may remove the need for these couple lines.  As far as I am aware, currently booleans are, when sent to XSLT, set to `true` if true and `""` if false - were you having issue with that?

Reply:
Ahh I was confused before I think because I was seeing an empty <repeatedQuote> tag and I misunderstood, but that makes sense if false is set to the empty string. I'll fix this.